andela/travel_tool_front
# #162197071 Super admin can edit user role

#### What does this PR do?

A super admin should be able to edit user roles

#### Description of Task to be completed?
- Super admin is able to click on `edit` and a pop-up modal pops up
- Super admin is able to edit the role and save the changes
#### How should this be manually tested?

- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Change directory - `cd travel_tool_front`
- Checkout to current branch - `git checkout ft-super-admin-edit-user-roles-162197071`
- Run application - `yarn start`
- Login into the application
- Assign yourself the role of super admin in the database
- Click on settings, then user role
- Make sure that you use [this backend](https://github.com/andela/travel_tool_back/pull/198)
- Try edit the user role

#### Any background context you want to provide?

- You must be a super admin to perform this operation

#### What are the relevant pivotal tracker stories?

[#162197071](https://www.pivotaltracker.com/story/show/162197071)

#### Screenshots (if appropriate)

<img width="925" alt="screen shot 2018-11-29 at 17 00 36" src="https://user-images.githubusercontent.com/32410518/49226711-6e229800-f3f8-11e8-94ca-50102a865393.png">

#### Questions:
